generators/pythonv2/codegen/src/ast/PythonFile.ts

```diff
@@ -20,7 +20,7 @@ export class PythonFile extends AstNode {
     public readonly moduleName: string;
     public readonly path: string[];
     public readonly name: string;
-    private readonly statements: (Class | Method | CodeBlock)[] = [];
+    private readonly statements: AstNode[] = [];
```

Comment:
Unrelated change I'm sneaking in. You should be able to add any sort of node as a statement to a file imo.

Reply:
nice